src/core/models.py

```diff
@@ -6,12 +8,12 @@
 class BaseModel(db.Model):
     __abstract__ = True
 
-    id = db.Column(db.Integer(), autoincrement=True, primary_key=True)
+    id = db.Column(db.UUID(), primary_key=True, default=uuid.uuid4)
     created_at = db.Column(db.DateTime(), default=datetime.now)
     updated_at = db.Column(db.DateTime(), default=datetime.now, onupdate=datetime.now)
 
-    def save(self, **kwargs):
-        model = self.__class__(name=self.name)
-        db.session.add(model)
-        db.session.commit()
-        return model
+#    def save(self, **kwargs):
```

Comment:
@PedroTeixeira-dev We can run some tests to try to make this work. Leave this code here for future tests.

Reply:
ok